src/hypermedia-constants.js

```diff
@@ -242,7 +242,8 @@ export const Classes = {
 	},
 	associations: {
 		singleAssociation: 'single-association',
-		potentialAssociation: 'potential-association'
+		potentialAssociation: 'potential-association',
+		defaultScoringRubricAssociation: 'default-scoring-rubric-association'
```

Comment:
This class still going to be used?

Reply:
Ah no this was for the other way I was attempting yesterday, will remove.